lib/Platform/Unicode/CharacterProperties.cpp

```diff
@@ -210,4 +211,123 @@ uint32_t canonicalize(uint32_t cp, bool unicode) {
   }
 }
 
+#ifdef HERMES_ENABLE_UNICODE_REGEXP_PROPERTY_ESCAPES
+
+/// Find a matching entry (such as \p NameMapEntry or \p RangeMapEntry) by
+/// matching a string \p name against the entry's \p name field.
+template <class T>
+const T *findNameMapEntry(
```

Comment:
Let's rename this to match the description, and mark it as static.

```suggestion
static const T *findMapEntry(
```

Reply:
Good point, I've done that with the commit that fixes up the logic flow in `unicodePropertyRanges`.